mmv1/templates/terraform/custom_flatten/netapp_volume_replicaton_mirror_state.go.erb

```diff
@@ -0,0 +1,20 @@
+func flatten<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
+  // Actual state of replication_enabled depends on mirrorState. let's update it.
+  // This is to pickup manual user STOP/RESUME operations on the replication.
+  if v == nil {
+    return v
+  }
+
+  if v.(string) == "STOPPED" {
```

Comment:
Do we know how long does it take for mirrorState to reach `STOPPED` after the stop action is taken? Will it be `STOPPED` immediately after the operation is done or is like reaching to `MIRRORED`?